weave

```diff
@@ -1197,6 +1202,9 @@ launch() {
                 IPRANGE="${1#*=}"
                 IPRANGE_SPECIFIED=1
                 ;;
+            --hairpin-mode=*)
+                HAIRPIN_MODE="${1#*=}"
```

Comment:
OK, let us raise it then. Should the default not be set to `true`? Should it not be uniform behaviour across the board, wherein `weave launch` also will run in hairpin mode unless disabled? That would mean putting it in `weaver`, which is the entrypoint to the `weaveworks/weave` container, right?

Or maybe we should leave it as 2 separate PRs...